clippy_lints/src/lib.rs

```diff
@@ -48,9 +48,9 @@ macro_rules! declare_clippy_lint {
     };
 }
 
-pub mod consts;
+mod consts;
 #[macro_use]
-pub mod utils;
+mod utils;
```

Comment:
This was used by rls:
https://github.com/rust-lang-nursery/rls/blob/3f3877613b58e1b592dde736dbf3dca48db2131a/src/build/rustc.rs#L164

Reply:
Does RLS use any `utils` anywhere else?

Reply:
I don't think so.